test/integration/instance_stats_test.go

```diff
@@ -2,52 +2,49 @@ package integration
 
 import (
 	"context"
+	"reflect"
 	"testing"
-	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/jarcoal/httpmock"
+	"github.com/linode/linodego"
 )
 
 func TestInstanceStats_Get(t *testing.T) {
-	// Skip on normal runs due to long-running test
-	t.Skip()
+	client := createMockClient(t)
 
-	client, instance, teardown, err := setupInstance(t, "fixtures/TestInstanceStats_Get")
-	defer teardown()
-	if err != nil {
-		t.Error(err)
+	desiredResponse := linodego.InstanceStats{
+		Title: "test_title",
+		Data: linodego.InstanceStatsData{
+			CPU: [][]float64{},
+			IO: linodego.StatsIO{
+				IO: [][]float64{},
+				Swap: [][]float64{},
+			},
+			NetV4: linodego.StatsNet{
+				In: [][]float64{},
+				Out: [][]float64{},
+				PrivateIn: [][]float64{},
+				PrivateOut: [][]float64{},
+			},
+			NetV6: linodego.StatsNet{
+				In: [][]float64{},
+				Out: [][]float64{},
+				PrivateIn: [][]float64{},
+				PrivateOut: [][]float64{},
+			},
+		},
```

Comment:
Could this be populated with some arbitrary data just to make sure everything unmarshals correctly?